Sources/TuistKit/Commands/EnvKey/ParseableArgument+Extension.swift

```diff
@@ -0,0 +1,221 @@
+import ArgumentParser
+import Foundation
+import TuistSupport
+
+extension Option {
```

Comment:
This is awesome! I'd consider opening an issue on the argument parser repository proposing this as a built-in feature of the package.